usgs-details.html

```diff
@@ -57,14 +74,16 @@
                 fallback: [
                     5.5, 5, 5.5, 6, 6.5, 7, 7.5, 8, 7.5, 7, 6.5, 6
                 ],
+                display: 'pH',
             },
             dissolvedOxygen: {
                 code: '00300',
                 units: 'mg/L',
                 transform: function(dO) { return dO; }, // Identity
                 fallback: [
                     10, 12, 10, 8, 6, 4, 2, 1, 2, 4, 6, 8
-                ]
+                ],
+                display: 'oxygen',
```

Comment:
I suppose having a separate label is potentially useful, even though we could have used the `METRICS` key for this (and renaming it as we see fit).